### PR TITLE
update intellij idea eap

### PR DESCRIPTION
**Pull request by gobinathm**
- Correcting the App Stanza

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

File: Casks/intellij-idea-eap.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   auto_updates true
 
-  app 'IntelliJ IDEA.app'
+  app 'IntelliJ IDEA 2017.1 EAP.app'
```
Comment (neutric) [inline Casks/intellij-idea-eap.rb]: This should be `app "IntelliJ IDEA #{version.before_comma} EAP.app"`.

Comment (gobinathm) [inline Casks/intellij-idea-eap.rb]: @neutric  Updated, thanks for the suggestion

**Comment by neutric** [comment]
On second thought: this cask probably should have remained the way it was before your edits in #3446. Since the current release for IntelliJ IDEA is 2016.3.5, `intellij-idea-eap.rb` and `intellij-idea-ce-eap.rb` should remain concerned with 2016.3 EAPs. The 2017.1 EAPs are in `intellij-idea-ce-eap.rb` and  `intellij-idea-next-ce-eap.rb`, last updated in #3440.

Although there might be no more 2016.3 EAPs after this one, I think this should be updated to the latest version as per https://confluence.jetbrains.com/display/IDEADEV/IDEA+2016.3+EAP to fix the problem in https://github.com/caskroom/homebrew-cask/issues/30873

**Comment by metyl** [comment]
Thx @neutric, yes we should keep `intellij-idea-eap` referring current release (2016.3) and keep `intellij-idea-next-eap` for next release (2017.1).

**Comment by gobinathm** [comment]
Closing this PR in favor of #3453

**Pull request closed without merging**